### ISSUE TITLE
Stack Raster Tifs issues - providing a dir instead of a path to a file.tif 

### ISSUE THREAD

**Issue by lwasser** [earthlab/earthpy#148]
Just a note for @joemcglinchy 
i'm happy to work on this with you but... here is something a student just tried to do!! :) 

```
I'm having a very interesting issue with trying to stack rasters that I'm hoping someone can help me with.

I have downloaded tiff files from Digital Globe and am now trying to open them in my local Jupyter Notebook. I've made a quick function called "get_bands" to glob all the bands for a single location at a single date together. I've omitted the documentation because it would make this submission too long.

FUNCTION

def get_bands(location, date):
bands = glob("data/group/digital-globe/" + location + "/" + date + "/*.tif")
return bands

----------------------------------------------------------

CODE

test = get_bands("hotel", "06_06_2018")
test.sort()

OUTPUT

['data/group/digital-globe/hotel/06_06_2018\\dg_06_06_2018_band_0_hotel.tif',
 'data/group/digital-globe/hotel/06_06_2018\\dg_06_06_2018_band_1_hotel.tif',
 'data/group/digital-globe/hotel/06_06_2018\\dg_06_06_2018_band_2_hotel.tif',
 'data/group/digital-globe/hotel/06_06_2018\\dg_06_06_2018_band_3_hotel.tif']
 

Then I try to do es.stack_rasters with a path to outputs:

test_path = "data/group/digital-globe/outputs/"

es.stack_raster_tifs(test, test_path)

 

This gives me the following error:

CPLE_OpenFailedError: Attempt to create new tiff file 'data/group/digital-globe/outputs/' failed: Permission denied
```

**Comment by joemcglinchy** [comment]
closed with PR #170 and #156. @lwasser can you please close.

**Comment by lwasser** [comment]
Thank you @joemcglinchy  closing this.
